handbook/company/pricing-features-table.yml

```diff
@@ -723,18 +717,8 @@
   usualDepartment: IT
   productCategories: [Device management]
   pricingTableCategories: [Device management]
-- industryName: Deploy security agents on macOS, Windows, and Linux computers.
-  documentationUrl: https://github.com/fleetdm/fleet/issues/14921
-  description: Easily configure and install SentinelOne, Crowdstrike, and other security tools.
-  moreInfoUrl: https://github.com/fleetdm/fleet/issues/14921
-  tier: Premium
-  jamfProHasFeature: appleOnly
-  jamfProtectHasFeature: no
-  usualDepartment: IT
-  productCategories: [Device management]
-  pricingTableCategories: [Device management]
-- industryName: Update apps on macOS, Windows, and Linux computers.
-  description: Update Zoom, Slack, Chrome, and other apps without interrupting your end users.
+- industryName: Application management
```

Comment:
Changed to **"Application management"** and added OS information to the description/tooltip.